packages/components/src/utils/rtl.js

```diff
@@ -3,6 +3,11 @@
  */
 import { css } from '@emotion/core';
 
+const lowerLeftRegExp = new RegExp( /-left/g );
+const lowerRightRegExp = new RegExp( /-right/g );
+const upperLeftRegExp = new RegExp( /Left/g );
+const upperRightRegExp = new RegExp( /Right/g );
```

Comment:
They look like constants to me.

https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#constants

```suggestion
const LOWER_LEFT_REGEXP = new RegExp( /-left/g );
const LOWER_RIGHT_REGEXP = new RegExp( /-right/g );
const UPPER_LEFT_REGEXP = new RegExp( /Left/g );
const UPPER_RIGHT_REGEXP = new RegExp( /Right/g );
```